packages/sdk/src/utils/index.ts

```diff
@@ -0,0 +1,369 @@
+import * as ecc from "@bitcoinerlab/secp256k1";
+import { BIP32Interface } from "bip32";
+import {
+  crypto,
+  initEccLib,
+  Network as BitcoinNetwork,
+  networks,
+  Payment,
+  PaymentCreator,
+  payments,
+  Psbt,
+  Signer,
+  Transaction,
+} from "bitcoinjs-lib";
+import { Buffer } from "buffer";
+import ECPairFactory, { ECPairInterface } from "ecpair";
+
+import { ADDRESS_TYPE_TO_FORMAT } from "../addresses/constants";
+import type { AddressFormat, AddressType } from "../addresses/types";
+import type { Network } from "../config/types";
+import { OrditSDKError } from "../errors";
+import type { UTXO } from "../transactions/types";
+import type {
+  BufferOrHex,
+  EncodeDecodeObjectOptions,
+  GetScriptTypeResponse,
+  IsBitcoinPaymentResponse,
+  NestedObject,
+  OneOfAllDataFormats,
+} from "./types";
+
+export function getNetwork(value: Network) {
+  if (value === "mainnet") {
+    return networks.bitcoin;
+  }
+
+  return networks[value];
+}
+
+export function createTransaction(
```

Comment:
Good time to rename this fn to `generatePayment` or `createPayment`. 

`createTransaction` is super misleading.

Reply:
Renamed